graphenex/core/web/__init__.py

```diff
@@ -53,5 +54,8 @@ def run_server(args=None, exit_shell=True):
         socketio.stop()
         if exit_shell:
             Shell().do_exit(None)
+    except socket.error as e:
+        if e.errno == 98:
+            logger.error('Looks like the specified addr:port is already in use!')
```

Comment:
Can we maybe use an error message like "`localhost:8000 is already in use!`" to show whichever address/port is being used?